package.json

```diff
@@ -1,8 +1,82 @@
 {
-  "name": "@hostmakerco/example-npm-publish",
-  "version": "1.0.11",
-  "main": "index.js",
-  "repository": "git@github.com:hostmakerco/example-npm-publish.git",
-  "author": "Nikas Praninskas <nikaspran@gmail.com>",
-  "license": "UNLICENSED"
+  "name": "pusher-chatkit-gifted-chat",
+  "version": "1.0.0",
+  "description": "Common components for zendesk",
+  "author": "AlexHayton",
+  "license": "MIT",
+  "repository": "hostmakerco/zendesk-apps",
+  "main": "dist/index.js",
+  "module": "dist/index.es.js",
+  "jsnext:main": "dist/index.es.js",
+  "types": "dist/index.d.ts",
+  "engines": {
+    "node": ">=8",
+    "npm": ">=5"
+  },
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "dev": "rollup -c -w",
+    "start": "REACT_APP_MOCK_ZENDESK=true yarn dev",
+    "build": "rollup -c",
+    "build:typescript": "tsc -p tsconfig.declarations.json",
+    "lint": "eslint --cache --ext .js,.jsx,.ts,.tsx ./",
+    "ci:lint": "eslint --ext .js,.jsx,.ts,.tsx . --format junit --output-file ./reports/eslint/result.xml",
+    "test": "react-scripts-ts test --env=jsdom",
+    "ci:test": "cross-env JEST_JUNIT_OUTPUT=./reports/jest/result.xml CI=1 react-scripts-ts test --env=jsdom --reporters=default --reporters=jest-junit",
+    "prepare": "yarn build"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "yarn lint && yarn test"
+    }
+  },
+  "dependencies": {
+    "@pusher/chatkit-client-react": "0.1.2",
+    "lodash-es": "^4.17.11",
```

Comment:
This is a fairly heavy library, do we actually need it? If it's just one or two methods we should implement them ourselves.

Reply:
Alternatively, let's depend on the individual packages for the functions that we need.

Reply:
@nikaspran tree shaking should reduce it?

Reply:
I will do some checks on the rollup output to confirm

Reply:
Hmmm, but it's still in dependencies, so it will always be fetched by apps using this library. I'd also look into if treeshaking actually fires here, it's usually a bit more tricky to set up